pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1800,8 +1800,6 @@ func (dm *DockerManager) computePodContainerChanges(pod *api.Pod, podStatus *kub
 				continue
```

Comment:
nit: Deleting the `if` statement and keeping the switch case seem cleaner to me.